gsod.md

```diff
@@ -53,8 +53,8 @@ The project itself will take approximately six months to complete, this would be
 | Budget item   | Amount | Running Total | Notes/justifications |
 | ------------------- | ------------------- | --------------| ----------- |
 | Technical writer/s audit, update, test, and publish new Mautic documentation    |  10,500    | 10,500.00 | |
-| Volunteer stipends  |  500      | 12,000 | 3 volunteer stipends x 500 each |
-| Project t-shirts (5 t-shirts)   | 13,000    |1300 | Design and print T-Shirt for participants and volunteers|
+| Volunteer stipends  |  500      | 12,000.00 | 3 volunteer stipends x 500 each |
```

Comment:
```suggestion
| Volunteer stipends  |  1,500      | 12,000 | 3 volunteer stipends x 500 each |
```

Thanks for picking this up - actually it should be 1500 (3x 500) and the running total therefore 12,000 so I've tweaked that in my suggestion above.